RecoTracker/LST/plugins/alpaka/LSTModulesDevESProducer.cc

```diff
@@ -0,0 +1,55 @@
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/ESProducer.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/ModuleFactory.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+
+#include "RecoTracker/Record/interface/TrackerRecoGeometryRecord.h"
+
+// LST includes
+#include "RecoTracker/LSTCore/interface/alpaka/Module.h"
+#include "RecoTracker/LSTCore/interface/alpaka/LST.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  class LSTModulesDevESProducer : public ESProducer {
+  public:
+    LSTModulesDevESProducer(const edm::ParameterSet& iConfig);
+
+    static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+
+    std::unique_ptr<SDL::LSTESHostData<SDL::Dev>> produceHost(TrackerRecoGeometryRecord const& iRecord);
```

Comment:
Could all the framework-facing code use CMSSW-defined type aliases for Alpaka types? I.e.
```suggestion
    std::unique_ptr<SDL::LSTESHostData<Device>> produceHost(TrackerRecoGeometryRecord const& iRecord);
```